metecho/api/gh.py

```diff
@@ -139,8 +139,7 @@ def get_project_config(**kwargs):
     """
     Expects to be in a local_github_checkout.
     """
-    global_config = GlobalConfig()
-    return global_config.get_project_config(**kwargs)
+    return ProjectConfig({}, **kwargs)
```

Comment:
@wlonk @davisagli This does not seem to be returning the merged config, combining the local project with the global default.

```
Traceback (most recent call last):
  File "/app/metecho/api/jobs.py", line 774, in available_task_org_config_names
    {"key": key, **value} for key, value in config.orgs__scratch.items()
AttributeError: 'NoneType' object has no attribute 'items'
```

Reply:
The first arg when constructing a ProjectConfig should be a GlobalConfig instance, not a dict.